client/state/data-layer/wpcom/sites/memberships/index.js

```diff
@@ -57,6 +55,23 @@ export const handleMembershipsList = dispatchRequestEx( {
 	onError: noop,
 } );
 
+export const handleSubscribedMembershipsList = dispatchRequestEx( {
+	fetch: action =>
+		http(
+			{
+				method: 'GET',
+				path: '/me/memberships/subscriptions',
+			},
+			action
+		),
+	onSuccess: ( {}, { subscriptions, total } ) => ( {
+		type: MEMBERSHIPS_SUBSCRIPTIONS_RECEIVE,
+		subscriptions,
+		total,
+	} ),
+	onError: noop,
+} );
 export default {
-	[ MEMBERSHIPS_PRODUCTS_LIST ]: [ handleMembershipsList ],
+	[ MEMBERSHIPS_PRODUCTS_LIST ]: [ handleMembershipProductsList ],
+	[ MEMBERSHIPS_SUBSCRIPTIONS_LIST ]: [ handleSubscribedMembershipsList ],
 };
```

Comment:
we want to break these files into different places and paths, one at `state/data-layer/wpcom/sites/memberships/products/index.js` and the other at `state/data-layer/wpcom/me/memberships/subscriptions/index.js`

Reply:
Done!